httpony/server.py

```diff
@@ -9,9 +9,16 @@
 
 
 def main():
+    """Read arguments if given"""
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-l", "--listen", help="Set ip/hostname for HTTPony to use",
```

Comment:
I think you can safely drop `for HTTPony to use`. "set the IP or hostname" would be my recommendation.